core/embed/rust/src/ui/model_tt/layout.rs

```diff
@@ -312,24 +310,19 @@ extern "C" fn new_confirm_action(n_args: usize, args: *const Obj, kwargs: *mut M
 
 fn _confirm_blob(
     title: StrBuffer,
-    data: Option<StrBuffer>,
-    description: Option<StrBuffer>,
-    extra: Option<StrBuffer>,
+    data: Obj,
+    description: Obj,
+    extra: Obj,
```

Comment:
The disadvantage of this approach is that it's _too_ flexible.
We want `data` to be `str | bytes`, but `description: str` and `extra: str`.

Reply:
de9937d